packages/svelte/src/internal/client/render.js

```diff
@@ -1280,6 +1280,10 @@ function handle_event_propagation(root_element, event) {
 	const handled_at = event.__root;
 	if (handled_at) {
 		const at_idx = path.indexOf(handled_at);
+		if (at_idx !== -1 && root_element === document) {
+			// This is the fallback document listener but the event was already handled -> ignore
```

Comment:
Need to check how the case of multiple Svelte instances is handled - they shouldn't cause double invocations

Reply:
Works